response_operations_ui/exceptions/exceptions.py

```diff
@@ -5,6 +5,20 @@ def __init__(self, response):
         self.message = response.text
 
 
+class ExternalApiError(ApiError):
+    def __init__(self, response=None, error_code=None):
+        self.response = response
+        self.error_code = error_code
+        if response:
+            self.url = response.url
+            self.status_code = response.status_code
+            self.message = response.text
+        else:
+            self.url = None
+            self.status_code = None
+            self.message = None
```

Comment:
There might not be a response object if the connection failed to be made. Probably want to add the `url` in a param in those cases.